File: go/darts/darts.go
package darts

// Score ...
func Score(x, y float64) int {
	switch r := x*x + y*y; {
	case r <= 1:
		return 10
	case r > 1 && r <= 25:
		return 5
	case r > 25 && r <= 100:
		return 1
	}
	return 0
}

File: python/darts/darts.py
def score(x, y):
    if x * x + y * y <= 1:
        return 10
    elif x * x + y * y <= 25 and x * x + y * y > 1:
        return 5
    elif x * x + y * y <= 100 and x * x + y * y > 25:
        return 1
    else:
        return 0

